owa/app/components/__tests__/admissionLocation/rightPanel/bedLayout/__snapshots__/bedLayout-test.js.snap

```diff
@@ -4,6 +4,16 @@ exports[`BedLayout Should render bed block properly 1`] = `
 <div
   className="bed-layout"
 >
+  <div
+    className="location option"
+  >
+    <label
+      className="button"
+      onClick={[Function]}
+    >
+      Change Bed Layout
```

Comment:
@sanishmaharjan can you change this to 'Edit bed layout'?
Also we need a 'Delete bed layout' button.